tools/snp.sh

```diff
@@ -1228,6 +1325,7 @@ main() {
 
       setup_and_launch_guest
       wait_and_retry_command verify_snp_guest
+      wait_and_retry_command verify_platform_snp_bit_status guest
```

Comment:
This should maybe replace verify_snp_guest. They are essentially doing the same thing, the only difference is that the original function is checking dmesg, while the bit_status is looking at the guest MSR to verify enablement. No need to run both of them if the MSR check is working correctly.

Reply:
Ok, I commented verify_snp_guest above